app/views/tags/index.html.erb

```diff
@@ -16,12 +16,19 @@
         <thead>
           <tr class="table-header">
             <th>Name</th>
+            <th></th>
           </tr>
         </thead>
         <tbody>
           <% @tags.each do |tag| %>
             <tr class="t-tag">
               <td><%= tag.name %></td>
+              <td><%= link_to 'Delete',
```

Comment:
That's cool with me. Although I'm going to go the whole 🐷 and rename them everywhere.